src/upgradeDeps.js

```diff
@@ -53,11 +51,32 @@ module.exports = function upgradeDeps(dependencies, version, options = {}) {
     dependencies['@babel/core'] = version;
   }
 
+  const webpack = semver.coerce(dependencies.webpack);
+  const depsWebpack1 = webpack && webpack.major === 1;
+
   // Adds preset-flow if needed, especially since it was split out of
   // preset-react
-  if (hasFlow && !dependencies['@babel/preset-flow']) {
+  if (options.hasFlow && !dependencies['@babel/preset-flow']) {
     dependencies['@babel/preset-flow'] = version;
   }
 
+  // Later versions of babel-loader are incompatible with Webpack v1.
+  // https://github.com/babel/babel-loader/issues/505
+  if (depsWebpack1 && dependencies['babel-loader']) {
+    console.log('Updating babel-loader to v7.1.1 as this project uses Webpack v1');
```

Comment:
Maybe we should say, please update webpack if you can 😄 https://webpack.js.org/guides/migrating/? don't have to